GliaWidgets/Sources/Coordinators/Chat/ChatCoordinator.swift

```diff
@@ -28,6 +28,7 @@ class ChatCoordinator: SubFlowCoordinator, FlowCoordinator {
     private var filePickerController: FilePickerController?
     private var quickLookController: QuickLookController?
     private let environment: Environment
+    private let shouldStartWithTranscript: Bool
```

Comment:
With this, do you refer to the secure conversations chat transcript? Because the regular chat also has the possibility of displaying transcript, correct? but if you attempt to send a message, then it goes to live chat, not to secure conversations. Thus this variable needs to have something it its name that makes it clear that we are talking about secure conversations, and not live chat.

Reply:
Makes sense. Updated 186b258e236383df3f386e26ef840b8761d23706.